analyzers/src/SonarAnalyzer.Common/Facade/ILanguageFacade.cs

```diff
@@ -41,5 +41,6 @@ public interface ILanguageFacade<TSyntaxKind> : ILanguageFacade
         SyntaxFacade<TSyntaxKind> Syntax { get; }
         ISyntaxKindFacade<TSyntaxKind> SyntaxKind { get; }
         ITrackerFacade<TSyntaxKind> Tracker { get; }
+        bool IsAnyKind(SyntaxNode node, params TSyntaxKind[] syntaxKinds);
```

Comment:
This should be in `SyntaxFacade` type, right next to the existing `IsAnyKind` overload.